public/locales/en/app.json

```
@@ -27,7 +27,8 @@
     "setPinning": "Set pinning",
     "submit": "Submit",
     "unpin": "Unpin",
-    "unselectAll": "Unselect all"
+    "unselectAll": "Unselect all",
+    "exportDag": "Export DAG"
```

All docs in ecosystem refer to CARs, maybe it is better to rename this to:
```suggestion
    "exportDag": "Download as CAR"
```